src/runner_metrics.py

```diff
@@ -50,19 +50,32 @@ class PostJobStatus(str, Enum):
     """The status of the post-job phase of a runner."""
 
     NORMAL = "normal"
+    ABNORMAL = "abnormal"
```

Comment:
Can't we use `auto` for this enum ?

Reply:
[auto](https://docs.python.org/3.9/library/enum.html#using-automatic-values) will inserts integers instead of strings. One could override ` _generate_next_value_` but not sure if it is worth it.

Reply:
Yes I agree, let's keep it as it is